pkg/heartbeat/heartbeat.go

```diff
@@ -89,12 +89,12 @@ func New(
 
 // ID returns an ID generated from the heartbeat data.
 func (h Heartbeat) ID() string {
-	var branch string
+	branch := "unset"
```

Comment:
Is `"unset"` a special keyword when json marshaling?

Reply:
No, it's just used for identifying a heartbeat when storing offline.

Reply:
I don't understand... do we swap `"unset"` with `nil` somewhere before sending to the api?

Reply:
The `ID` function returns the unique identification for heartbeat. It's only used to identify offline heartbeats like a primary key for the database. To avoid having empty strings when the field is nil I changed to "unset".

Current
```
"1592868313.541149-file-coding---/tmp/main.go-false"
```
After
```
"1592868313.541149-file-coding-unset-unset-/tmp/main.go-false"
```

Reply:
Both current and after are the same if only used for identifying a heartbeat right?

Reply:
yes, now we won't see empty strings for branch nor project.

Reply:
Nevermind, that code is running in `ID` method so it's only used there not actually setting heartbeat attributes to `"unset"`. Ignore my questions above :)